integration_tests/package.json

```diff
@@ -10,7 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "axios": ">=0.21.2"
+    "axios": ">=1.1.3"
```

Comment:
It turns out that newer versions of axios will add the `Accept-Encoding` header to the http requests. Therefore we should update the minimum version here.  We also don't want to set the version at a fixed number since this can potentially reveal issues with newer versions of axios.